source/assets/stylesheets/styles/_project_story_large_tile.scss

```diff
@@ -0,0 +1,95 @@
+.project-story-large-tile {
+  box-sizing: border-box;
+  display: block;
+  margin-bottom: 6px;
+  padding-right: 6px;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    margin-bottom: 0;
+    width: 66%;
+  }
+}
+
+.project-story-large-tile__container {
+  background-color: $project-story-tile__background-color;
+  box-sizing: border-box;
+  border: 1px solid $project-story-tile__border-color;
+  padding: 6px;
+  width: 100%;
+
+  &:hover {
+    background-color: $project-story-tile__hover-color;
+  }
+}
+
+.project-story-large-tile__image-container {
+  box-sizing: border-box;
+}
+
+.project-story-large-tile__image {
+  display: block;
+  height: 125px;
+  object-fit: cover;
+  width: 100%;
+
+  @include breakpoint($max-mobile-container) {
+    height: 180px;
+  }
```

Comment:
Oh yes, the reason for adding so many heights was a way for fitting as much as possible content on mobile (hence smaller height on smaller resolutions). But you're right it doesn't scale well. 
We're gonna have new images so they will fit better, but I'm gonna take your advice and leave the images ratio as they are :)